modules/bloxone_infra_host_azure/userdata.tftpl

```diff
@@ -0,0 +1,7 @@
+#cloud-config
```

Comment:
```suggestion
# host setup configuration
```

Reply:
updated...

Reply:
This has been reverted back to `#cloud-config`. This specific values is required as per the documentation, updating this to any other value breaks the cloud-init provisioning.